Add article on download and feed refresh error (#174)
AntennaPod/antennapod.github.io: _i18n/en/documentation/bugs-first-aid/refresh-download-errors.md

> Do you get a notification saying that downloads completed with errors? Then AntennaPod could not retrieve the feed with episode information or an audio files from the website of the podcast hoster.

Reviewer: ```suggestion
Do you get a notification saying that downloads completed with errors? Then AntennaPod could not retrieve the feed with episode information or an audio file from the website of the podcast hoster.
```

The word "feed" is very technical. I would try to avoid it, or at least add "RSS". (AntennaPod supports other feed formats as well but I think RSS is what one sometimes sees on websites)

Issues that only occur with one or a few specific podcasts are often related to those podcasts. To confirm if this is the case, you can follow these steps:
1. tap on the notification to open the Download Log

> 2. tap on the specific notification with the error

Reviewer: Notification? I would rather call this "log entry" or simply "line"

> 3. long-press on the URL that is displayed, and copy it or open it in your browser

Reviewer: ```suggestion
3. long-press on the URL that is displayed, and copy it or open it in your web browser
```

> If the URL doesn't work in the browser either, this means that it is not a problem with the app. If you get the error message consistently then the podcast has likely been moved, taken off-line or not properly configured.

Reviewer: ```suggestion
If the URL doesn't work in the browser either, this means that it is definitely not a problem with AntennaPod. If you get the error message consistently then the podcast has likely been moved, taken off-line or not properly configured. Try contacting the podcast creators about it, as this is not something we could change on AntennaPod's end.
```

> Does the error message in the Download log say 'Failed to connect to [url]'? Then this means that something is interfering with your network. If you have an ad-blocker, VPN, captive portal, proxy, privacy guard, special DNS service, or parental control set up on your phone, you can try turning these off.

Reviewer: I think all error messages caused by apps messing with the network connection already say so in the error message. A generic error could be caused by something else. Not sure.

Author: Right. Well, let's keep it then. Can always adjust it if we get more info in future. Wouldn't know what/how to write differently.

> If you can't pinpoint the issue, please share a screenshot of the error together with the podcast URL (RSS feed) [on our forum](https://forum.antennapod.org/c/support/7). Our community will happily have a look to find out the exact cause!

Reviewer: No screenshots please. People should copy the text instead (there is a button for that). This avoids having to re-type everything, or information being hidden by scrolling. Also, it can be searched for.

Author: A screenshot would help in a [case like this](https://mstdn.io/@andreyka/108753257031053301) and the user doesn't figure out how to copy text there. But that would propbably come out still. Will adjust the text.

Do you get a notification saying that downloads completed with errors? Then AntennaPod could not refresh the podcast or download an audio file from the website of the podcast hoster.

Issues that only occur with one or a few specific podcasts are often related to those podcasts. To confirm if this is the case, you can follow these steps:
1. Tap on the notification to open the Download Log
2. Tap on the specific log entry with the error
3. Long-press on the URL that is displayed, and copy or open it in your web browser

If the URL doesn't work in the browser either, this means that it is not a problem with AntennaPod. If you get the error message consistently then the podcast has likely been moved, taken off-line or is not properly configured. In this case, please contact the podcast creators (as this cannot be fixed at our end).

Does the error message in the Download log say "Failed to connect to [url]"? Then this probably means that there is interference on your network. If you have an ad-blocker, VPN, captive portal, proxy, privacy guard, special DNS service, or parental control set up on your phone, you can try turning these off.

> Do you see an error message like "not well-formed (invalid token)”? Then the publisher messed up their (RSS) feed. (For example, they might have used a plain `&`-sign rather than encoding it as `&amp;`.) 

Reviewer: ```suggestion
Do you see an error message like "not well-formed (invalid token)”? Then the publisher messed up their (RSS) feed. (For example, they might have used a plain `&`-sign rather than encoding it as `&amp;`.) Please contact the publisher directly, as this cannot be fixed at our end.
```
